Code/Tools/AssetProcessor/native/AssetManager/ProductAsset.cpp

```diff
@@ -63,6 +64,9 @@ namespace AssetProcessor
             wasRemoved = AZ::IO::SystemFile::Delete(m_absolutePath.c_str());
         }
 
+        // Try to delete the metadata file too if one exists
```

Comment:
Nit: May worth checking for the return value to make sure that the file is deleted successfully.

Reply:
Added